src/main/java/edu/nps/moves/dis7/SignalPdu.java

```diff
@@ -105,16 +105,17 @@ public long getSampleRate() {
         return sampleRate;
     }
 
+    /**
+     * IAW IEEE 1278.1, this field shall specify the number of **bits** of digital
+     * voice audio or digital data being sent in this Signal PDU.
+     */
     public short getDataLength() {
-        return (short) data.length;
+        if (dataLength == 0) {
+            return (short) (data.length * 8);
```

Comment:
@adamwrobinson5 Could you explain the importance of checking length for 0 and the line that follows? 

We didn't do that on the DIS 6 equivalent Signal PDU and wondering if it's required.

Reply:
Not required, it was just a way to abstract a DIS-ism. IAW the standard, the data length should be the length of the data field in bits. Since data is a byte array, the data length field would be the length of the byte array multiplied by 8.

I can take it out though if you'd prefer it not be in there.

Reply:
Ok yes let's take it out and just rely on returning the dataLength.